amrit-docker-setup/nginx/nginx.conf

```diff
@@ -0,0 +1,309 @@
+        try_files $uri $uri/ /inventory/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /inventory {
+        return 301 /inventory/;
+    }
+    
+    location /tm/ {
+        alias /usr/share/nginx/html/tm/;
+        try_files $uri $uri/ /tm/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /tm {
+        return 301 /tm/;
+    }
+    
+    location /mmu/ {
+        alias /usr/share/nginx/html/mmu/;
+        try_files $uri $uri/ /mmu/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /mmu {
+        return 301 /mmu/;
+    }
+    
+    location /scheduler/ {
+        alias /usr/share/nginx/html/scheduler/;
+        try_files $uri $uri/ /scheduler/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /scheduler {
+        return 301 /scheduler/;
+    }
+    
+    location /hwc-scheduler/ {
+        alias /usr/share/nginx/html/hwc-scheduler/;
+        try_files $uri $uri/ /hwc-scheduler/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /hwc-scheduler {
+        return 301 /hwc-scheduler/;
+    }
+    
+    location /hwc-inventory/ {
+        alias /usr/share/nginx/html/hwc-inventory/;
+        try_files $uri $uri/ /hwc-inventory/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /hwc-inventory {
+        return 301 /hwc-inventory/;
+    }
+    
+    location /ecd/ {
+        alias /usr/share/nginx/html/ecd/;
+        try_files $uri $uri/ /ecd/index.html;
+        add_header Cache-Control "no-cache";
+        add_header X-Content-Type-Options "nosniff";
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        index index.html;
+    }
+    location /ecd {
+        return 301 /ecd/;
+    }
+    
+    # location /helpline104/ {
```

Comment:
We can add it back.
I am able to build these with Node 12.
Also let's name it /104 or /1097